spec/spec_helper.rb

```diff
@@ -10,3 +10,19 @@ module SpecHelper
 end
 
 WebMock.disable_net_connect!(allow: 'coveralls.io')
+
+# Executes the provided block after adjusting the ENV to have the
+# provided keys and values set as defined in hash. After the block
+# completes, restores the ENV to its previous state.
+def with_env_values(hash)
+  old_vals = ENV.select { |k, v| hash.include?(k) }
```

Comment:
Sadly `ENV` has some of the methods of a hash, but is actually an `Object` :frowning: